src/index.css

```diff
@@ -225,3 +224,24 @@ body {
   justify-content: center;
```

Comment:
- [ ] It would be better if we implement the CSS DRY principle, there are a couple of repeating CSS rules. I would highlight a few here but I urge you to thoroughly go through and fix them. `font-family: 'Roboto', sans-serif; font-weight: 300; font-weight: bold; text-decoration: none; font-family: 'Montserrat', 'Times New Roman', Times, serif; display: flex;`

Reply:
Thank You for pointing this out, indeed that was some code repetition in my index file, this issue is solved now.